packages/form-builder/src/blocks/fields/amount/inspector/index.tsx

```diff
@@ -29,7 +29,8 @@ const Inspector = ({attributes, setAttributes}) => {
                     <CurrencyControl
                         label={__('Set Donation', 'give')}
                         value={setPrice}
-                        onValueChange={(setPrice) => setAttributes({setPrice})}
+                        onBlur={() => setPrice || setAttributes({setPrice: 100})}
```

Comment:
I just checked legacy and our default value there is 25.  Can we update to that instead so at least it matches current functionality?

<img width="771" alt="Screenshot 2023-03-28 at 4 21 45 PM" src="https://user-images.githubusercontent.com/10138447/228357788-70a6bd28-af96-48d8-b942-a6de0fda93e8.png">

Reply:
Resolved, see https://github.com/impress-org/givewp-next-gen/pull/145/commits/c776c8797807b40ab7d33d2a615e7f5beddc7970.